### DISCUSSION TITLE
is there debian version ?

### DISCUSSION THREAD

**Discussion by realbitt** [dvershinin/fds#25] [General]
is there debian version ?

**Comment by dvershinin** [comment]
There's no Debian version. Maybe someone could create a similar utility using ufw instead of
firewalld.

**Comment by mdpstreet** [comment]
Hi Danila,

Thank you for your work with FDS. Appreciate the work you have done with it. I was hoping there might be some solution for AlmaLinux 9. This was a downstream Centos 9 project. 

Regards,
Mark Street

**Reply by dvershinin** [reply]
@mdpstreet sure, just added the EL9 build of the packages, this would work with AlmaLinux 9. Check out the updated README.

**Comment by mdpstreet** [comment]
Hi Danila,

Thanks so much for responding so fast. Was not expecting that quick 😊 . I will be giving it try sometime today.

Regards,

Mark Street


From: Danila Vershinin ***@***.***>
Reply to: dvershinin/fds ***@***.***>
Date: Thursday, 17 November 2022 at 8:07 pm
To: dvershinin/fds ***@***.***>
Cc: Mark Street ***@***.***>, Mention ***@***.***>
Subject: Re: [dvershinin/fds] is there debian version ? (Discussion #25)


@mdpstreet<https://github.com/mdpstreet> sure, just added the EL9 build of the packages, this would work with AlmaLinux 9. Check out the updated README.

**Comment by mdpstreet** [comment]
Hi Danila,

Running fine on AlmaLinux9

[root@ip-10-0-0-182 /]# fds --version
fds 0.0.35

Thanks again,

Mark Street
